src/igvfd/schemas/treatment.json

```diff
@@ -0,0 +1,206 @@
+{
+    "title": "Treatment",
+    "id": "/profiles/treatment.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "treatment_term_name",
+        "treatment_type",
+        "treatment_term_id",
+        "amount",
+        "duration"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        },
+        {
+            "$ref": "mixins.json#/aliases"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/notes"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        },
+        {
+            "$ref": "mixins.json#/product_info"
+        }
+    ],
+    "dependentSchemas": {
+        "amount": {
+            "required": [
+                "amount_units"
+            ],
+            "comment": "Amount units must be specified if amount is specified"
+        },
+        "duration": {
+            "required": [
+                "duration_units"
+            ],
+            "comment": "Duration units must be specified if duration is specified"
+        },
+        "treatment_term_id": {
+            "oneOf": [
+                {
+                    "properties": {
+                        "treatment_type": {
+                            "enum": [
+                                "protein"
+                            ]
+                        },
+                        "treatment_term_id": {
+                            "pattern": "^((UniProtKB:[A-Z0-9]{6})|(NTR:[0-9]{2,8}))$"
+                        }
+                    }
+                },
+                {
+                    "properties": {
+                        "treatment_type": {
+                            "enum": [
+                                "chemical"
+                            ]
+                        },
+                        "treatment_term_id": {
+                            "pattern": "^((CHEBI:[0-9]{1,7})|(NTR:[0-9]{2,8}))$"
+                        }
+                    }
+                }
+            ]
+        }
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "documents": {
+            "description": "Documents that describe the treatment protocol."
+        },
+        "amount": {
+            "title": "Amount",
+            "type": "number"
+        },
+        "amount_units": {
```

Comment:
please add description, and probably comment explaining that units go with amount